azure/scope/clients.go

```diff
@@ -47,23 +45,23 @@ func (c *AzureClients) CloudEnvironment() string {
 
 // TenantID returns the Azure tenant id the controller runs in.
 func (c *AzureClients) TenantID() string {
-	return c.Values[auth.TenantID]
+	return c.Values["AZURE_TENANT_ID"]
```

Comment:
I feel like I should technically point out that constants for these would be good, but since the SDK doesn't even expose these that I can find, I'm not gonna push to do that here.

Reply:
Yes, it's frustrating that there aren't any constants for these in the SDK. We could define them locally, but I didn't see a good common package to house them at first glance.